project/AppDependencies.scala

```diff
@@ -4,21 +4,21 @@ object AppDependencies {
 
   import play.core.PlayVersion
 
-  private val silencerVersion = "1.7.9"
-  private val bootstrapPlayVersion = "6.3.0"
-  private val hmrcMongoPlayVersion = "0.68.0"
+  private val silencerVersion      = "1.7.9"
+  private val bootstrapPlayVersion = "6.4.0"
```

Comment:
On all other project you update to 7.2 would be good to align the same here:
[info]   uk.gov.hmrc:bootstrap-frontend-play-28  : 6.4.0             -> 7.2.0
[info]   uk.gov.hmrc:bootstrap-test-play-28:test : 6.4.0             -> 7.2.0